File: ReverseString.java
class Solution {
    public void reverseString(char[] s) {
        for(int i=0; i<=(s.length-1)/2; i++){
            char ch = s[i];
            s[i] = s[s.length-1-i];
            s[s.length-1-i] = ch;
        }
    }
}

File: ReverseString.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
 // TC: O(N) SC: O(1)
var reverseString = function(s) {
    const n = s.length - 1;
    for(let i=0; i<=Math.floor(n / 2); i++) {
        const tempChar = s[i];
        s[i] = s[n - i];
        s[n - i] = tempChar;
    }
};
